### ISSUE TITLE
Classic Editor doesn't automatically fetch and display the X.com tweet in a embedded format and shows up just a plain URL

### ISSUE THREAD

**Issue by pavel-lipcean** [WordPress/classic-editor#216]
Hello,

An increasing number of writers and editors are opting for the new x.com URLs over the older twitter.com URLs to embed them within their posts (at least within our network). However, unlike Twitter URLs, x.com URLs currently remain as plain links and do not embed. Below is a screenshot comparing how the older twitter URLs appear versus the new X.com URLs.

![06-24-24-Classic-Editor-Twitter-X](https://github.com/WordPress/classic-editor/assets/5729949/71c5514f-7364-420a-b989-213b843c07a0)

Important, writers can't directly access the old Twitter URLs because they now redirect to the new x.com. The URLs can only be retrieved by choosing "Embed post" after clicking the "More options" button https://publish.twitter.com/?url=https://twitter.com/WordPress/status/1803123544672395641#. Essentially, all links may soon transition to x.com.

Please consider adding support for embedding the new X.com URLs alongside the older twitter.com URLs.

**Comment by i0n1ca** [comment]
Hello Pavel,
I have the same issue with x and twitter URLs, it works if you put manually twitter.com instead of x.com
Today I have contacted X Support and provided the same arguments as yours and I am waiting for a response to see if X related or Classic Editor. https://x.com/i0n1ca/status/1815305822341091553
ionica

**Comment by ironprogrammer** [comment]
Thanks for the report, @pavel-lipcean! This oembed functionality is actually part of WordPress Core (the plugin only re-enables the classic editor), so I'll close this issue since it's being tracked on [#59142: X/Twitter posts cannot be embedded](https://core.trac.wordpress.org/ticket/59142).

*tl;dr* on the status, we're waiting to see when X/Twitter settles on the oembed endpoint, which is currently under `twitter.com` 🙃. The results from testing the embeds changes periodically, which makes confidence in a Core fix more challenging.
